### PR TITLE
Improved headerBackTitle and headerTruncatedBackTitle docs.

### PR DESCRIPTION
**Pull request by leonelpena**
#190

### CODE REVIEW

File: docs/stack-navigator.md [hunk 1]
```diff
@@ -95,11 +95,48 @@ React Element or Component to display custom image in header's back button. When
 
 #### `headerBackTitle`
 
-Title string used by the back button on iOS, or `null` to disable label. Defaults to the previous scene's `headerTitle`.
+Title string used by the back button on iOS, or `null` to disable label. Defaults to the previous scene's `headerTitle`. `headerBackTitle` has to be defined in the origin screen not in the destination screen. For instance, when you have a transition A to B and you want to disable the `headerBackTitle` on `B`:
```
Comment (vonovak) [inline docs/stack-navigator.md]: it's a small thing, but let's make this right: there is a comma missing: `in the origin screen not in the destination screen` should be `in the origin screen, not in the destination screen`.

Same thing applies to the next paragraph.

Comment (leonelpena) [reply docs/stack-navigator.md]: sure!

**Comment by vonovak** [comment]
thanks!

**Comment by leonelpena** [comment]
You are welcome!

**Comment by brentvatne** [comment]
deployed, thanks!

**Pull request merged**